obsidian.el

```diff
@@ -497,8 +498,10 @@ See `markdown-follow-link-at-point' and
     context))
 
 (defun obsidian--mention-link-p (match)
-  "Check if MATCH produced by `obsidian--grep' is a link."
-  (obsidian--link-p (obsidian--elgrep-get-context match)))
+  "Check if MATCHES produced by `obsidian--grep' contain a link."
+          (mapcar (lambda (element) 
+                    (if (listp element) 
```

Comment:
You can also replace `if` with `when` here, since it doesn't have an else branch

Reply:
Hi, I’ve fixed the code according to your comments. It should be ok now.

Unfortunately I can’t test and lint it with eldev right now since I’m not currently on emacs 27.2 and I can’t upgrade right now… so if you can test and lint it after merge it would be great.

thanks a lot
Dave

Reply:
Hi, everything is linted now and I also improved the functionality so that if you list backlinks of a file the file itself is not listed.